python_modules/dagster-graphql/dagster_graphql/implementation/events.py

```diff
@@ -369,6 +369,8 @@ def from_dagster_event_record(event_record: EventLogEntry, pipeline_name: str) -
             logKey=data.file_key,
             stepKeys=data.step_keys,
             externalUrl=data.external_url,
+            externalStdoutUrl=data.external_stdout_url or data.external_url,
+            externalStderrUrl=data.external_stderr_url or data.external_url,
```

Comment:
maybe we should pick one as the Default one rather than using the same URL for both STDOUT and STDERR? maybe just send it to STDOUT in that case?